src/main/java/seedu/address/logic/messages/HelpMessages.java

```diff
@@ -11,20 +11,40 @@ public class HelpMessages extends Messages {
             "Opened help window for add command.";
     public static final String MESSAGES_SHOWING_SEARCH_HELP_MESSAGE =
             "Opened help window for search command.";
+    public static final String MESSAGES_SHOWING_EXIT_HELP_MESSAGE =
```

Comment:
just a suggestion for future reference when cleaning up code (if we are ever going to do it), for neatness you could consolidate all `MESSAGES_SHOWING_XXX_HELP_MESSAGE` to a single constant with value `"Opened help window for %1$s command." and abstract the command names as enums. Not sure if it would be overkill for TP but good coding conventions to have.